app-doc/heirloom-doctools/heirloom-doctools-999999.ebuild

```diff
@@ -0,0 +1,81 @@
+EAPI=6
+
+inherit git-r3 flag-o-matic toolchain-funcs readme.gentoo-r1
+
+DESCRIPTION="Classic Unix documentation tools ported from OpenSolaris"
+HOMEPAGE="https://n-t-roff.github.io/heirloom/doctools.html"
+EGIT_REPO_URI="git://github.com/n-t-roff/heirloom-doctools.git"
+
+LICENSE="CDDL"
+SLOT="0"
+IUSE="cxx"
+
+RDEPEND="!sys-apps/groff"
+DEPEND="sys-devel/flex
+	virtual/yacc"
+
+src_prepare() {
+	# mpm uses C++, we'll build it explicitly if we really want to
+	sed -i -e 's:mpm:$(MPM):' makefile || die
+
+	eapply_user
```

Comment:
please use `default` instead of `eapply_user`, and do it as the first thing in `src_prepare`. And even better: replace the `sed` here too